x-pack/test_serverless/functional/test_suites/observability/screenshot_creation/response_ops_docs/cases/list_view.ts

```diff
@@ -59,5 +63,29 @@ export default function ({ getPageObject, getPageObjects, getService }: FtrProvi
       await navigateToCasesApp(getPageObject, getService, owner);
       await svlCommonScreenshots.takeScreenshot('cases', screenshotDirectories, 1700, 1024);
     });
+
+    it('case detail screenshot', async () => {
+      await pageObjects.common.navigateToUrlWithBrowserHistory(
```

Comment:
Just to be sure that we indeed navigated to the case detail page I think we should a) wait for all loading to finish and b) make sure that at least the title of the case is visible. You can do that by:

```
await pageObjects.header.waitUntilLoadingHasFinished();
await testSubjects.existOrFail('case-view-title');
```

Reply:
Thanks! Done!